kotlinpoet/gradle.properties

```diff
@@ -2,3 +2,4 @@ POM_ARTIFACT_ID=kotlinpoet
 POM_NAME=KotlinPoet
 POM_DESCRIPTION=Use beautiful Kotlin code to generate beautiful Kotlin code.
 POM_PACKAGING=jar
+kotlin.mpp.applyDefaultHierarchyTemplate=false
```

Comment:
Any issues with adopting the default hierarchy template vs opting out?

Reply:
Since nonJvm is not the default hierarchy, we don't use 'default hierarchy'. 
If we don't add this property, a warning will appear at `build`:

```
The Default Kotlin Hierarchy Template was not applied to 'project ':kotlinpoet'':
Explicit .dependsOn() edges were configured for the following source sets:
[jsMain, nonJvmMain, wasmJsMain]

Consider removing dependsOn-calls or disabling the default template by adding
    'kotlin.mpp.applyDefaultHierarchyTemplate=false'
to your gradle.properties

Learn more about hierarchy templates: https://kotl.in/hierarchy-template
```

Reply:
Makes sense!